website/releases/release-1.1.0.md

```diff
@@ -0,0 +1,366 @@
+
+For detailed technical information, see the [PR documentation](https://github.com/apache/hudi/pull/13650/).
+
+### Removal of Auto-Commit Support in WriteClient
+
+For developers using `HoodieWriteClient` directly, auto-commit support has been removed. The `hoodie.auto.commit` configuration is no longer honored.
+
+**Migration Required**:
+
+```java
+// Previous (with auto-commit)
+HoodieWriteConfig config = // instantiate config
+SparkRDDWriteClient writeClient = new SparkRDDWriteClient(engineContext, config);
+String instantTime = writeClient.startCommit();
+writeClient.upsert(JavaRDD<HoodieRecord>, instantTime);
+
+// Now Required (explicit commit)
+HoodieWriteConfig config = // instantiate config
+SparkRDDWriteClient writeClient = new SparkRDDWriteClient(engineContext, config);
+String instantTime = writeClient.startCommit();
+JavaRDD<WriteStatus> writeStatus = writeClient.upsert(RDD<HoodieRecord>, instantTime);
+writeClient.commit(instantTime, writeStatus); // Explicit commit now mandatory
+```
+
+This change applies to table services as well.
+
+### INSERT INTO Behavior Change
+
+The default behavior of Spark SQL's `INSERT INTO` command has changed. Previously, it used "upsert" operation for tables with precombine/ordering fields, which caused deduplication. From 1.1.0, `INSERT INTO` now performs "insert" operation by default, ingesting records as-is without deduplication.
+
+**Example**:
+
+```text
+Commit1: 
+  Partition1, recordKey1, val1, orderingValue1
+  Partition1, recordKey1, val2, orderingValue2
+
+Pre-1.1.0: Returns one record (based on ordering field)
+From 1.1.0: Returns both records
+```
+
+**To Restore Previous Behavior**: Set `hoodie.spark.sql.insert.into.operation = upsert`
+
+### Restriction on Multiple Bulk Inserts for COW with Simple Bucket
+
+Multiple `bulk_insert` operations are now restricted for Copy-On-Write tables when **all** of the following conditions are met:
+
+1. Table type is COW
+2. Index type is `BUCKET` with engine set to `SIMPLE`
+3. Spark native row writer is disabled (`hoodie.datasource.write.row.writer.enable = false`)
+
+**Resolution**: Use `upsert` operation after the initial `bulk_insert`, or enable Spark native row writer.
+
+### Flink Bucket Index Restriction
+
+Bucket index now supports only UPSERT operations and cannot be used with append write mode in Flink.
+
+---
+
+## Behavior Changes
+
+### Default Merge Mode Changed
+
+The default record merging behavior has changed based on whether a precombine field is set:
+
+- **No Precombine Field**: Defaults to `COMMIT_TIME_ORDERING` using `OverwriteWithLatestAvroPayload` (last-write-wins)
+- **Precombine Field Set**: Defaults to `EVENT_TIME_ORDERING` using `DefaultHoodieRecordPayload` (classic event-time based comparison)
+
+The default value for `hoodie.datasource.write.precombine.field` (previously `ts`) has been removed.
```

Comment:
This option is deprecated, we now prefer to use table config option: `hoodie.table.ordering.fields`.